tooling-tests/src/test/java/io/ballerina/protoc/tools/ToolingPackageTest.java

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.protoc.tools;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static io.ballerina.protoc.tools.ToolingTestUtils.GENERATED_SOURCES_DIRECTORY;
+import static io.ballerina.protoc.tools.ToolingTestUtils.assertGeneratedSources;
+
+/**
+ * gRPC packaging related tests.
+ */
+public class ToolingPackageTest {
+
+    @Test(enabled = false)
+    public void testSimplePackage() {
+        try {
+            Files.createDirectories(Paths.get(GENERATED_SOURCES_DIRECTORY, "tool_test_packaging_1"));
+        } catch (IOException e) {
+            Assert.fail("Could not create target directories", e);
+        }
+        assertGeneratedSources("packaging", "tool_test_packaging_1/simplePackage.proto",
+                "simplePackage_pb.bal", "helloworld_service.bal", "helloworld_client.bal",
+                "tool_test_packaging_1");
+    }
+
+    @Test(enabled = false)
+    public void testPackagingWithMessageImports() {
+        try {
+            Files.createDirectories(Paths.get(GENERATED_SOURCES_DIRECTORY, "tool_test_packaging_2"));
+        } catch (IOException e) {
+            Assert.fail("Could not create target directories", e);
+        }
+        assertGeneratedSources("packaging", "tool_test_packaging_2/packageWithMessageImport.proto",
+                "packageWithMessageImport_pb.bal", "helloworld_service.bal",
+                "helloworld_client.bal", "tool_test_packaging_2", "modules/message/message_pb.bal");
+    }
+
+    @Test(enabled = false)
```

Comment:
Is this intended?

Reply:
Added in [6d4d80b](https://github.com/ballerina-platform/protoc-tools/pull/3/commits/6d4d80b05617ed62f2910e14b60e0d1dc4177cca)

Reply:
Enabled in [29fbdf8](https://github.com/ballerina-platform/protoc-tools/pull/3/commits/29fbdf828af4c4679fa5e3179ae9cf0330c97264)